pycomlink/validation/stats.py

```diff
@@ -25,8 +25,8 @@
                                      'coefficient_of_variation',
                                      'root_mean_square_error',
                                      'mean_absolute_error',
-                                     'R_mean_reference',
-                                     'R_mean_cml',
+                                     'R_sum_reference',
```

Comment:
The "problem" with the sum is that you cannot compare them for reference and predicted since there might be different amount of NaN and hence you would have to normalize. Not sure, if we should leave this to the "user"...

Reply:
For calculating the mean we, of course, have to take the issue with NaN-handling into account.

Reply:
I did not change the calculation of` R_mean_cml/R_sum_reference`, but only the name, to fit the calculation. 
For the calculation of the mean: Only non-nan pairs are taken into account, therefore both `R_mean_reference` and `R_mean_predicted` are comparable.

Reply:
Okay. But there is no `R_mean_reference` and `R_mean_predicted` in the `RainError`. Should be added.